pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -231,22 +231,30 @@ func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, req
 		return err
 	}
 
-	localCM := &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      nameAdditionalTrustedCA(instance),
-			Namespace: instance.Namespace,
-		},
-		Data: sourceCM.Data,
-	}
+	if _, ok := sourceCM.Data[NameCertConfigMapKey]; ok {
```

Comment:
```suggestion
	if _, ok := sourceCM.Data[NameCertConfigMapKey]; !ok {
		reqLogger.Info("Found ConfigMap referenced by ImageConfig.Spec.AdditionalTrustedCA.Name but did not find key 'cincinnati-registry' for registry CA cert.", "Name", image.Spec.AdditionalTrustedCA.Name, "Namespace", openshiftConfigNamespace)
		return nil
	}
	...		
```

Reply:
Should we return an `error` in this scenario?

Reply:
My thought was not to return an error because this would be the case where a cluster has something in `ImageConfig.Spec.AdditionalTrustedCA.Name`, but the cincinnati-operator doesn't care about it.  I figured I would log a message in case a user did intend for Cincinnati to get the cert, but used the wrong key.

Reply:
Maybe a good opportunity to use a condition in the status.

Reply:
Condition could be a follow-up PR in case you want to get this merged and make an issue to track the condition.

Reply:
https://github.com/openshift/cincinnati-operator/issues/29

Reply:
I think my comment about returning an error may have taken attention away from my actual comment. Instead of this huge `if { ... } else { log message }` just invert the okay and `return nil` if that's what you want to do.

Reply:
Inverted it.  Let me know if I misunderstood the suggestion.